app/src/main/res/layout/subscription_item.xml

```diff
@@ -1,55 +1,97 @@
+        app:cardElevation="0dp">
 
-    <com.joanzapata.iconify.widget.IconTextView
-        android:id="@+id/txtvTitle"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_alignStart="@+id/imgvCover"
-        android:layout_alignLeft="@+id/imgvCover"
-        android:layout_alignTop="@+id/imgvCover"
-        android:layout_alignEnd="@+id/imgvCover"
-        android:layout_alignRight="@+id/imgvCover"
-        android:layout_alignBottom="@+id/imgvCover"
-        android:background="@color/non_square_icon_background"
-        android:ellipsize="end"
-        android:gravity="center"
-        android:textColor="?android:attr/textColorPrimary"
-        tools:text="@string/app_name" />
-
-    <de.danoeh.antennapod.ui.common.TriangleLabelView
-        android:id="@+id/triangleCountView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_alignParentTop="true"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        app:backgroundColor="?attr/colorSecondary"
-        app:corner="rightTop"
-        app:primaryText="Test"
-        app:primaryTextColor="?attr/colorOnSecondary"
-        app:primaryTextSize="12sp" />
+        <LinearLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <androidx.cardview.widget.CardView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:clickable="false"
+                app:cardCornerRadius="8dp"
+                app:cardElevation="0dp">
+
+                <RelativeLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content">
+
+                    <de.danoeh.antennapod.ui.common.SquareImageView
+                        android:id="@+id/imgvCover"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:scaleType="fitCenter"
+                        android:outlineProvider="background"
+                        squareImageView:direction="width"
+                        tools:src="@tools:sample/avatars" />
+
+                    <TextView
+                        android:id="@+id/countViewPill"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="3"
+                        android:layout_alignParentEnd="true"
+                        android:textSize="12sp"
+                        style="@style/TextPill"
+                        app:primaryText="Test"
+                        app:primaryTextColor="?attr/colorOnSecondary"
+                        app:primaryTextSize="12sp" />
```

Comment:
I have never seen this attributes used on a TextView. Are you sure that they are necessary?